### DISCUSSION TITLE
How to hide the menu summary

### DISCUSSION THREAD

**Discussion by kaisiant** [facebook/docusaurus#7938] [Q&A] [answered]
I don't want to show the menu summary, how do I configure it? Thanks.
![image](https://user-images.githubusercontent.com/26655111/184071260-eb144619-1d89-4ad3-84f9-8d3e067bba4f.png)

**Comment by slorber** [comment, marked as answer]
The "category generated index" is not displayed by default. 

If it is displayed, it's because you explicitly added configuration for it.

https://docusaurus.io/docs/sidebar/items#generated-index-page
